sdk/core/azure-core/test/e2e/azure_core_storage_test_sample.cpp

```diff
@@ -5,9 +5,9 @@
 #define _CRT_SECURE_NO_WARNINGS
 #endif
 
-#include "azure/core/http/curl/curl.hpp"
-#include "azure/core/http/http.hpp"
-#include "azure/core/http/pipeline.hpp"
+#include <azure/core/http/curl/curl.hpp>
+#include <azure/core/http/http.hpp>
+#include <azure/core/http/pipeline.hpp>
```

Comment:
So, for tests we need <> and for non tests "" ?

Reply:
Yes, that mirrors the way our customers are most likely to include our headers

> Updated tests to mirror how a customer will include the header
> 
> #include <>